src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -133,15 +135,17 @@ public BigInteger UnclaimedGas(StoreView snapshot, UInt160 account, uint end)
         }
 
         [ContractMethod(0_05000000, ContractParameterType.Boolean, ParameterTypes = new[] { ContractParameterType.PublicKey }, ParameterNames = new[] { "pubkey" })]
```

Comment:
I think that the cost of this proposal should be higher

Reply:
Don't worry about the fee. We will adjust the economic model later.